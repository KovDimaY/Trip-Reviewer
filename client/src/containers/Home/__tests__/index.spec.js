import React from 'react';
import { create } from 'react-test-renderer';
 
import HomeContainer from './../../Home';
import Root from './../../../hoc/root';

const mockComponent = props => {
    return (
        <Root>
            <HomeContainer {...props} />
        </Root>
    );
};
 
describe('<HomeContainer />', () => {
    it('should render component', () => {
        const props = {
            trips: {}
        };
        const tree = create(mockComponent(props)).toJSON();
        
        expect(tree).toMatchSnapshot();
    });
});