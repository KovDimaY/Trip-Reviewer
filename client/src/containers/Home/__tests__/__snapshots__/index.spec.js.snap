// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeContainer /> should render component 1`] = `
<div>
  <div
    className="loadmore"
    onClick={[Function]}
  >
    Load More
  </div>
</div>
`;

exports[`<HomeContainer /> should render component with a list of items 1`] = `
<div>
  <TripItem
    _id="_id"
    author="author"
    duration="duration"
    price="price"
    rating="rating"
    title="title"
  />
  <div
    className="loadmore"
    onClick={[Function]}
  >
    Load More
  </div>
</div>
`;
